.github/workflows/deploy-to-ecs.yml

```
@@ -0,0 +1,74 @@
+env:
+  AWS_REGION: ap-northeast-2
+  ECR_REPOSITORY: tyquill-api
+  ECS_SERVICE: tyquill-api-service
+  ECS_CLUSTER: tyquill-cluster
+  TASK_DEFINITION_FAMILY: tyquill-api-task
+  CONTAINER_NAME: tyquill-api
+
+permissions:
+  id-token: write
+  contents: read
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)